### PR TITLE
Update FromAndriy

### PR DESCRIPTION
**Pull request by AndriiNaulychnyi**

### CODE REVIEW

**Comment by VasylNaulychnyi** [comment]
Looks like we made changes in the file 'FromAndriy' at the same time.

**Comment by AndriiNaulychnyi** [comment]
> Looks like we made changes in the file 'FromAndriy' at the same time.

Yes. I will resolve all conflicts.

File: FromAndriy [hunk 1]
```diff
@@ -1,3 +1,2 @@
-Test file
```
Comment (AndriiNaulychnyi) [inline FromAndriy]: Can I remove this line? @VasylNaulychnyi

Comment (VasylNaulychnyi) [reply FromAndriy]: @AndriiNaulychnyi Yes, you can.

**Pull request merged**